### PR TITLE
ci: send db-migration output to stdout rather than a file

### PR DESCRIPTION
**Pull request by lemmih**
## Summary of changes

Changes introduced in this pull request:

- Show DB migration results in the github-worker output. This makes it a bit easier to see when something goes wrong.
- The sync output is still hidden. You must re-run the code locally to see the errors if something goes wrong during sync.

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: scripts/tests/calibnet_db_migration.sh [hunk 1]
```diff
@@ -46,7 +46,10 @@ echo "data_dir = \"${TMP_DIR}/data_dir\"" >> "${CONFIG_FILE}"
 echo 'encrypt_keystore = false' >> "${CONFIG_FILE}"
 
 # Run the current Forest with the old database. This should trigger a migration (or several ones).
-forest --chain calibnet --encrypt-keystore false --log-dir "$LOG_DIRECTORY" --detach --save-token ./admin_token --track-peak-rss --config "${CONFIG_FILE}"
+forest --chain calibnet --log-dir "$LOG_DIRECTORY" --halt-after-import --save-token ./admin_token --track-peak-rss --config "${CONFIG_FILE}"
```
Comment (LesnyRumcajs) [inline scripts/tests/calibnet_db_migration.sh:49]:
```suggestion
forest --chain calibnet --log-dir "$LOG_DIRECTORY" --halt-after-import --track-peak-rss --config "${CONFIG_FILE}"
```

**Review by LesnyRumcajs — APPROVED** [review]

File: scripts/tests/calibnet_db_migration.sh (hunk 1 above)
Comment (LesnyRumcajs) [reply scripts/tests/calibnet_db_migration.sh:49]: it doesn't hurt but it's not really useful

File: scripts/tests/calibnet_db_migration.sh [hunk 2]
```diff
@@ -46,7 +46,10 @@ echo "data_dir = \"${TMP_DIR}/data_dir\"" >> "${CONFIG_FILE}"
 echo 'encrypt_keystore = false' >> "${CONFIG_FILE}"
 
 # Run the current Forest with the old database. This should trigger a migration (or several ones).
-forest --chain calibnet --encrypt-keystore false --log-dir "$LOG_DIRECTORY" --detach --save-token ./admin_token --track-peak-rss --config "${CONFIG_FILE}"
+forest --chain calibnet --log-dir "$LOG_DIRECTORY" --halt-after-import --track-peak-rss --config "${CONFIG_FILE}"
+
+# Sync to HEAD. This might reveal migrations errors not caught above.
+forest --chain calibnet --log-dir "$LOG_DIRECTORY" --detach --save-token ./admin_token --track-peak-rss --config "${CONFIG_FILE}"
```
Comment (elmattic) [inline scripts/tests/calibnet_db_migration.sh:52]: No need to use `sync wait` just after?

**Review by elmattic — APPROVED** [review]

File: scripts/tests/calibnet_db_migration.sh (hunk 2 above)
Comment (lemmih) [reply scripts/tests/calibnet_db_migration.sh:52]: The `sycn wait` is a few lines down.

**Pull request merged**